.gitignore

```diff
@@ -37,7 +37,8 @@ ipython_config.py
 .LSOverride
 
 # Icon must end with two \r
-Icon


```

Comment:
Can you take this part of the diff out? This line is deliberately formatted with `\r` which your editor probably didn't like when it saved :)

Reply:
I wasn't planning to merge this PR - Imma close it

Reply:
This was just a test PR for github settings : P
But ye u r correct it was prob my editor. I actually even erased my commit and re-did it when I first saw it but it just kept showing up